d2l-test-runner.config.js

```diff
@@ -1,10 +1,3 @@
-const pattern = type => `+(components|controllers|directives|helpers|mixins|templates)/**/*.${type}.js`;
-
 export default {
-	pattern,
-	groups: [{
-		name: 'aXe',
-		files: pattern('axe'),
-		browsers: ['chrome']
-	}]
+	pattern: type => `**/test/*.${type}.js`
```

Comment:
Are we sure there's no nested test files?

Reply:
Yeah, I compared the lists from `npm run test:* -- --watch`. We could still expand it, though.

Reply:
I just know we've accidentally not run all our tests before, but probably not an issue if you're adding _new_ tests. But maybe